README.md

```diff
@@ -15,11 +15,24 @@ See the difference by disabling this optimization using the `-0` option. Or just
 
 ## Installation
 
+### Standard installation
+
     sudo apt install imagemagick || yum install ImageMagick
     git clone https://github.com/stefanhaustein/TerminalImageViewer.git
     cd TerminalImageViewer/src/main/cpp
     make
     sudo make install
+    
+### Using Docker
+
+Just replace any command with:
+
+    docker run --rm -v <dir>:/home bperel/tiv
```

Comment:
How is bperel/tiv resolved? Does this mean you are hosting a copy? (sorry I am a docker noob) O:)

Reply:
When the `docker run` command is executed, Docker will download the Docker image called `bperel/tiv` (which contains the compiled `tiv` executable) if it doesn't exist locally, then use it to create a container containing tiv.

It would make more sense that the Docker image is hosted under your username than undermine. To do that : 
- Create an account named stefanhaustein (to make it consistent with your GitHub username) on https://hub.docker.com/
- Open your terminal and cd to your local repository
- Run `docker build . -t stefanhaustein/tiv`
- Run `docker push stefanhaustein/tiv`
- Now you can use for instance `docker run --rm -v ${PWD}:/home stefanhaustein/tiv test.png` to display an image called `test.png` and located in the same directory as your local repository.

Tell me once you've done that and I'll update my PR so that `stefanhaustein/tiv` is mentioned instead of `bperel/tiv`.

Tell me if I'm not clear :-)

Reply:
I think it would make more sense for me to create and maintain a snap, and I would like to avoid maintaining too many things/accounts. So for now, I think it would make sense to add some kind of "contributed installation packages" or similar to the readme, pointing to your docker and the SUSE RPM somebody just created: https://github.com/stefanhaustein/TerminalImageViewer/issues/33

Does this make sense?

Reply:
p.s. Thanks a lot for setting this up btw!

Reply:
> I think it would make more sense for me to create and maintain a snap, and I would like to avoid maintaining too many things/accounts. So for now, I think it would make sense to add some kind of "contributed installation packages" or similar to the readme, pointing to your docker and the SUSE RPM somebody just created: #33
> 
> Does this make sense?

Sure, that's fine by me. In that case, do you think that you could create a release on GitHub each time there is a new stable version? (if you intend to maintain this project further) Then I'll watch this repo for new releases (it's possible on Github since a few days) and push a new Docker image each time there is a new release.

Reply:
Yes, sounds good, will do!